src/libstore/globals.hh

```diff
@@ -638,6 +638,13 @@ public:
           can add it to `trusted-public-keys` in their `nix.conf`.
         )"};
 
+    Setting<Strings> remoteSigningUrls{
```

Comment:
At some point, we may want to specify (in docs somewhere) how these servers should work (with RFC-style `MUST`s littered everywhere :laughing: ), but of course that can wait until the idea / approach is approved / until _we_ know what we want the server to be able to do.

Reply:
Addressed in the doc commit, it's a reasonable yet simple attempt I believe for now. We can upgrade this in RFC style later on.